bin/Makefile_ROCm

```diff
@@ -0,0 +1,78 @@
+# Makefile for Cray (XT5, XE/XK, XC90) with Cray Compiling Environment (CCE)
+# If PrgEnv-cray is not the default, use "module swap" to PrgEnv-cray
+
+FORTRAN_FIXEDFORM    = -ffixed
+FORTRAN_BOUND_CHECK  = #-R bcp #-- FIXME: bound checking causes issue with
+                               #          OpenMP. Reported to Cray.
+FORTRAN_DEBUG        = -g -O0 #-hdevelop -eF -hfp0 -K trap=fp
+FORTRAN_OPTIMIZE     = -O1
+FORTRAN_PROFILE      = 
+
+CC_DEBUG             = -g
+CC_OPTIMIZE          = $(FORTRAN_OPTIMIZE)
+
+#-- Default and may be overwritten below
+DEVICE_HIP           = 1
+DEVICE_COMPILE       = 
+
+LINK = $(FORTRAN_LINK)
+
+# Define default switches for this compiler. These can be overwritten at
+# build time with, e.g., 'make ENABLE_OMP_OFFLOAD=0 <target>'
+ENABLE_OMP          ?= 1
+ENABLE_OMP_OFFLOAD  ?= 0
+USE_ISO_10646       ?= 0
+
+#-- Silo library may be  automatically included & linked by 
+#-- "module load silo" if provided, otherwise, manually put include and 
+#-- link line flag &  location below
+SILO_DIR ?= ${HOME}/localsw/poplar/silo/4.10.2_cce9.1.3
+INCLUDE_SILO = -I${SILO_DIR}/include
+LIBRARY_SILO = -L${SILO_DIR}/lib -lsilo
+
+#-- HDF5 library may be  automatically included & linked by 
+#-- "module load hdf5" if provided, otherwise, manually put include and 
+#-- link line flag &  location below
+HDF5_DIR ?= ${HOME}/localsw/poplar/hdf5/1.10.12_cce10
+INCLUDE_HDF5 = -I${HDF5_DIR}/include
+LIBRARY_HDF5 = -L${HDF5_DIR}/lib -lhdf5_fortran -lhdf5
+
+OPENMPI_DIR ?= ${HOME}/localsw/poplar/hdf5
+INCLUDE_OPENMPI = -I${OPENMPI_DIR}/lib
+LIBRARY_OPENMPI = -L${OPENMPI_DIR}/lib -lmpi_usempif08 -lmpi_usempi_ignore_tkr -lmpi_mpifh -lmpi
+
+#-- if empty, don't link to HDF5
+ifeq ($(strip $(HDF5_DIR)),)  
+  INCLUDE_HDF5 = 
+  LIBRARY_HDF5 = 
+endif
+
+INCLUDE_PREPROCESSOR = -I$(PATH_GENASIS)/Build
+
+#-- The following stanzas check whether we are building with CUDA or HIP
+ifeq ($(findstring nvidia,${CRAY_ACCEL_TARGET}), nvidia)
+  DEVICE_CUDA    = 1
+  DEVICE_COMPILE = nvcc -c
+  INCLUDE_DEVICE = -I${CUDA_ROOT}/include
+  LIBRARY_DEVICE = -L${CUDA_ROOT}/lib64 -lcudart
+endif
+
+ifeq ($(findstring amd,${CRAY_ACCEL_TARGET}), amd)
+  DEFINES       += -D__HIP_PLATFORM_HCC__
+  DEVICE_HIP     = 1
+  DEVICE_COMPILE ?= hipcc -c -D__HIP_PLATFORM_HCC__
+  INCLUDE_DEVICE = -I${ROCM_PATH}/include
+  LIBRARY_DEVICE = -L${ROCM_PATH}/lib -lamdhip64
+endif
```

Comment:
Remove all of this.  This Makefile is only for ROCm compiler.

Reply:
Done!